test/e2e/specs/classic-editor.test.js

```diff
@@ -24,4 +24,36 @@ describe( 'classic editor', () => {
 		const textEditorContent = await page.$eval( '.wp-editor-area', ( element ) => element.value );
 		expect( textEditorContent ).toEqual( 'Typing in classic editor' );
 	} );
+
+	it( 'Should have wpautop disabled on a post containing blocks', async () => {
+		// Click the Text mode.
+		await page.click( '#content-html' );
+
+		// Enter some block text.
+		const original = `<!-- wp:paragraph -->
+<p>Foo bar four five six</p>
+<!-- /wp:paragraph -->
+
+This is another set of text`;
+		await page.keyboard.type( original );
+
+		// Save the post so that TinyMCE loads with wpautop disabled.
+		await page.click( '#save-post' );
```

Comment:
@tofumatt Could I get your help debugging this e2e test failure? My intent is to save the Classic Editor post, and then inspect TinyMCE behavior after the post is reloaded. However, it seems the test is progressing before the post is reloaded.